### PR TITLE
Data Base added, there are some code cleanup that needs to be done, d…

### PR DESCRIPTION
**Pull request by OmerMajeed**
…atabase is working nevertheless.

### CODE REVIEW

**Comment by OmerMajeed** [comment]
Osama, please do the review of the changes

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -74,28 +74,33 @@
         <activity
             android:name=".Startup_Pages.login_page"
             android:label="IdeaMeter"
-            android:theme="@style/AppTheme.NoActionBar">
-        </activity>
+            android:theme="@style/AppTheme.NoActionBar"></activity>
```
Comment (osamasalahuddin) [inline app/src/main/AndroidManifest.xml:77]: What is the reason behind this change ? @OmerMajeed

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -74,28 +74,33 @@
...
             android:value="@integer/google_play_services_version" />
+
         <activity
             android:name=".Startup_Pages.about_idealojy"
             android:label="About Idealojy"
             android:theme="@style/AppTheme.NoActionBar" />
-
-        <activity android:name=".Startup_Pages.splash"
+        <activity
+            android:name=".Startup_Pages.splash"
```
Comment (osamasalahuddin) [inline app/src/main/AndroidManifest.xml:91]: Line number:93 All the strings in the xml files will be defined and accessed from res/values/strings.xml file.
Like this line should have been like this.
@OmerMajeed correct all these changes
- [ ] All string values referred from strings.xml

android:label=@string/app_name

File: app/src/main/java/com/example/dell/helloandroid/DataBase/Contact.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ * Created by Omer Majeed on 4/12/2016.
+ */
+public class Contact {
+
+    //private variables
+    int _id;
+    String _name;
+    String _phone_number;
+
+    // Empty constructor
```
Comment (osamasalahuddin) [inline app/src/main/java/com/example/dell/helloandroid/DataBase/Contact.java:13]: Comments to be used will be of the format "/ \* \* /" .
@OmerMajeed Kindly remove all the // comments to "/ \*  \* /".

**Comment by osamasalahuddin** [comment]
@OmerMajeed kindly do all the above suggested changes in the code. Otherwise the app is working fully
